modules/panorama/variables.tf

```diff
@@ -149,3 +149,20 @@ variable "metadata" {
   type        = map(string)
   default     = {}
 }
+
+variable "service_account" {
+  description = "IAM Service Account for running Panorama instance (just the email)"
+  default     = null
+  type        = string
+}
+
+variable "scopes" {
+  default = [
+    "https://www.googleapis.com/auth/compute.readonly",
+    "https://www.googleapis.com/auth/cloud.useraccounts.readonly",
+    "https://www.googleapis.com/auth/devstorage.read_only",
```

Comment:
Do the panorama need access to object storage ? or other scopes as well ?

Reply:
Removed panorama access to storage from the default scopes